besu/src/main/java/org/hyperledger/besu/cli/config/ProfileName.java

```diff
@@ -16,6 +16,16 @@
 
 /** Enum for profile names. Each profile corresponds to a configuration file. */
 public enum ProfileName {
+  /**
+   * The 'ENTERPRISE' profile. Corresponds to the 'profiles/enterprise-private.toml' configuration
```

Comment:
nitpicking but I think you could remove the "Corresponds to the xyz configuration file" bit

Reply:
done